qiime_studio/api/v1.py

```diff
@@ -107,7 +106,8 @@ def api_input_artifacts(plugin_name, workflow_name, input_name):
     workflow = plugin.workflows[workflow_name]
     input_type = workflow.signature.inputs[input_name][0]
     input_artifacts = []
-    artifact_paths = glob.glob(os.path.join(os.getcwd(), '*.qzf'))
+    path = request.args.get('path', os.getcwd())
```

Comment:
For some reason the path argument isn't being split into key/values, it's all lumped together like

```python
[1] API: ImmutableMultiDict([('path/Users/Develop/Developer/work/q2-types/q2_types/tests/data', '')])
```

So input artifacts aren't loading, as it reverts to just using the server `.cwd()`